docs/DEVELOPING.md

```diff
@@ -1,21 +1,22 @@
 # Developing Guidelines
 
-- [Installation](#installation-to-run-locally)
-- [Technologies](#technologies)
-- [File Structure](#file-structure)
-- [Developing](#developing)
-  - [Running locally outside of Docker container](#running-locally-outside-of-docker-container)
-  - [Running locally within a Docker container](#running-locally-within-a-docker-container)
-  - [Database Changes](#database-changes)
-    - [Migrations](#migrations)
-  - [Adding an API Route](#adding-an-api-route)
-  - [Adding to the frontend app](#adding-to-the-frontend-app)
-  - [Testing](#testing)
-  - [Installing new packages](#installing-new-packages)
-- [Code Formatting](#code-formatting)
-- [Tips](#tips)
-- [Misc](#misc)
-- [Production](#production)
+- [Developing Guidelines](#developing-guidelines)
+  - [Installation to run locally](#installation-to-run-locally)
+  - [Technologies](#technologies)
+  - [File Structure](#file-structure)
+  - [Developing](#developing)
+    - [Running locally outside a Docker container](#running-locally-outside-a-docker-container)
+    - [Running locally within a Docker container](#running-locally-within-a-docker-container)
+    - [Database changes](#database-changes)
+      - [Migrations](#migrations)
+    - [Adding an API Route](#adding-an-api-route)
+    - [Testing](#testing)
+    - [Installing new packages](#installing-new-packages)
+  - [Code Formatting](#code-formatting)
+  - [Tips](#tips)
+  - [Testing the Production Environment](#testing-the-production-environment)
+  - [Production](#production)
+  - [Misc](#misc)
```

Comment:
I got a new VSCode plugin that does some fancy shit with markdown, and it auto-fixed this glossary which is hella sick